fileHub/js-app/app/services/api-service/api-service.js

```diff
@@ -16,84 +17,121 @@ export class ApiService {
   }
 
   /**
-   * @typedef {Object} token
-   *
    * Authenticates user in FileHub application.
    * @param {string} email
    * @param {string} password
-   * @returns {Promise<token, ClientServerError|ServerError>}>}
+   * @returns {Promise<string, ClientServerError|ServerError>}>}
    */
   async logIn(email, password) {
     const response = await this._fetch('/login', {
       method: 'POST',
       body: JSON.stringify({email, password}),
     });
 
+    this._checkResponseOnClientOrServerError(response);
```

Comment:
Let's check for 401. In this case form should render a message that login or password are wrong.